integration_tests/support/commands.ts

```diff
@@ -26,9 +27,35 @@ Cypress.Commands.add('signInAsLegalSender', () => {
     cy.task('stubVerifyOneTimeCode')
 
     // Request a one time code - we dont need to know what it is, but we do need to have submitted the form on this page
-    RequestOneTimeCodePage.goToPage().submitFormWithValidEmailAddress('valid@email.address.cjsm.net')
+    const emailSentPage: EmailSentPage = RequestOneTimeCodePage.goToPage().submitFormWithValidEmailAddress(
+      EmailSentPage,
+      'valid@email.address.cjsm.net'
+    )
     // Submit the one time code
-    const emailSentPage = Page.verifyOnPage(EmailSentPage)
     emailSentPage.submitFormWithValidOneTimeCode()
   }
 })
+
+Cypress.Commands.add('signInAsSmokeTestLegalSender', () => {
```

Comment:
Abstraction to sign in the Smoke Test Legal Sender user using either magic link or OTC